src/components/Page.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { TodoAdder } from './adder/TodoAdder';
+import { TodoList } from './list/TodoList';
+
+export default function Page({
+  title, onChange, todos, onClickAdd, onClickComplete,
+}) {
+  return (
+    <div>
+      <h1>To-do</h1>
+      <TodoAdder title={title} onChange={onChange} onClickAdd={onClickAdd} />
```

Comment:
```suggestion
      <TodoAdder 
        title={title}
        onChange={onChange}
        onClickAdd={onClickAdd}
      />
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편인데 어떤것 같나요?
코드 가독성을 고려한 코드 정리 기준을 정해봐도 좋을 것 같아요!

Reply:
네 기환님 저도 동의합니다 :) 
IDE 자동 정렬 세팅이 안되어있던 상태였던것 같아요 :) 말씀하신대로 정리하겠습니다.